editor/find_in_files.cpp

```diff
@@ -88,9 +87,10 @@ static bool find_next(const String &line, String pattern, int from, bool match_c
 
 //--------------------------------------------------------------------------------
 FindInFiles::FindInFiles() {
-	_root_prefix = ROOT_PREFIX;
+	// TODO: Unhardcode this.
```

Comment:
Wait... you're hardcoding the `res://` root prefix by removing the `ROOT_PREFIX` define, but then you've added a comment saying to unhardcode it? I'm confused.

Reply:
The "unharcode" comment is for the filters right below it. This is the only place in the codebase that doesn't just write "res://" directly, if this would be in a define, it then should be in one accessible to everyone.

Reply:
Oh, huh. Alright. I didn't realize that, woops!